### ISSUE TITLE
Switch to GNU Affero license

### ISSUE THREAD

**Issue by deavmi** [grena/gruik#51]
Switch to GNU Affero license.

**Comment by grena** [comment]
Hi!

I don't get the difference with the actual license. I read that:

> (...) specifically designed to ensure cooperation with the community in the case of network server software. (From https://opensource.org/licenses/AGPL-3.0)

Do you know some use case? Thanks!

**Comment by deavmi** [comment]
Well if you ran this on a server as GPL then you could make source code changes to the PHP without releasing the chnaged source code. However with the AGPL you would be required to release the changed source code. ~If I am right. This site explains it simply I guess - http://choosealicense.com/

**Comment by heyakyra** [comment]
yeah, the AGPL basically ensures that the reasons you would choose GPL are actually still going to be followed

**Comment by deavmi** [comment]
Yeah. It would be good for this case.

On 14 June 2016 11:06:32 PM SAST, "Kẏra" notifications@github.com wrote:

> yeah, the AGPL basically ensures that the reasons you would choose GPL
> are actually still going to be followed 
> 
> ---
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/grena/gruik/issues/51#issuecomment-226016341

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.
